source/hdev/Camera/Basic/CaptureHDRLoop.hdev

```diff
@@ -85,43 +87,20 @@
-<l>    get_dict_tuple (acquisitions, 0, firstAcquisition)</l>
-<l>    get_dict_tuple (firstAcquisition, 'Acquisition', acquisition)</l>
-<l>    get_dict_tuple (acquisition, 'ExposureTime', exposure_time)</l>
-<l>    set_framegrabber_param (AcqHandle, 'ExposureTime', exposure_time)</l>
-<l>    get_dict_tuple (acquisition, 'Aperture', aperture)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Aperture', aperture)</l>
-<l>    get_dict_tuple (acquisition, 'Brightness', brightness)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Brightness', brightness)</l>
-<l>    get_dict_tuple (acquisition, 'Gain', gain)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Gain', gain)</l>
-<c>    </c>
-<c>    * Getting settigns for second acqusition</c>
-<l>    get_dict_tuple (acquisitions, 1, secondAcquisition)</l>
-<l>    get_dict_tuple (secondAcquisition, 'Acquisition', acquisition)</l>
-<l>    get_dict_tuple (acquisition, 'ExposureTime', exposure_time)</l>
-<l>    set_framegrabber_param (AcqHandle, 'ExposureTime', exposure_time)</l>
-<l>    get_dict_tuple (acquisition, 'Aperture', aperture)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Aperture', aperture)</l>
-<l>    get_dict_tuple (acquisition, 'Brightness', brightness)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Brightness', brightness)</l>
-<l>    get_dict_tuple (acquisition, 'Gain', gain)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Gain', gain)</l>
-<c>    </c>
-<c>    * Getting settings for third acqusition</c>
-<l>    get_dict_tuple (acquisitions, 2, thirdAcquisition)</l>
-<l>    get_dict_tuple (thirdAcquisition, 'Acquisition', acquisition)</l>
-<l>    get_dict_tuple (acquisition, 'ExposureTime', exposure_time)</l>
-<l>    set_framegrabber_param (AcqHandle, 'ExposureTime', exposure_time)</l>
-<l>    get_dict_tuple (acquisition, 'Aperture', aperture)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Aperture', aperture)</l>
-<l>    get_dict_tuple (acquisition, 'Brightness', brightness)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Brightness', brightness)</l>
-<l>    get_dict_tuple (acquisition, 'Gain', gain)</l>
-<l>    set_framegrabber_param (AcqHandle, 'Gain', gain)</l>
-<c>    </c>
+<c>    * Getting acquisition settings</c>
+<l>    for Index := 0 to |AllKeys| - 1 by 1</l>
+<l>        get_dict_tuple (acquisitions, Index, tempAcquisition)</l>
+<l>        get_dict_tuple (tempAcquisition, 'Acquisition', acquisition)</l>
+<l>        get_dict_tuple (acquisition, 'ExposureTime', exposure_time)</l>
+<l>        set_framegrabber_param (AcqHandle, 'ExposureTime', exposure_time)</l>
+<l>        get_dict_tuple (acquisition, 'Aperture', aperture)</l>
+<l>        set_framegrabber_param (AcqHandle, 'Aperture', aperture)</l>
+<l>        get_dict_tuple (acquisition, 'Brightness', brightness)</l>
+<l>        set_framegrabber_param (AcqHandle, 'Brightness', brightness)</l>
+<l>        get_dict_tuple (acquisition, 'Gain', gain)</l>
+<l>        set_framegrabber_param (AcqHandle, 'Gain', gain)</l>
```

Comment:
As a separate commit, I suggest you change the format for all variable names in this sample to 'Aperture' 'Gain' etc. (capital first letter)

Reply:
Done